cg/models/demultiplex/flow_cell.py

```diff
@@ -23,11 +23,11 @@
 LOG = logging.getLogger(__name__)
 
 
-class FlowCell:
-    """Class to collect information about flow cell directories and there particular files."""
+class FlowCellDirectoryData:
```

Comment:
```suggestion
class FlowCellInformation:
```

Reply:
perhaps a bit better "DirectoryData" seems that it holds data for directories

Reply:
I think I prefer `FlowCellDirectoryData`, even though it is a clunky name it is true